go.mod

```diff
@@ -290,6 +290,9 @@ require (
 )
 
 replace (
+
+	// TODO MUST BE REMOVED IN THE NEXT FOLLOWING PR (or this PR needs to be split and cosmossdk.io/api tagged)
+	cosmossdk.io/api => ./api
```

Comment:
I think we should split the PR and tag `cosmossdk.io/api` to avoid introducing replaces